### DISCUSSION TITLE
Python Barrier/LeapInput Client

### DISCUSSION THREAD

**Discussion by jzhvymetal** [input-leap/input-leap#1895] [Show and tell]
Sorry for the duplicated post if you monitor Barrier repository. Here is Barrier client written in Python. I needed to learn Python, so what better way than to create a new project? All details can be found in the following repository link below.

https://github.com/jzhvymetal/PY_Barrier

FYI, I also wrote an Arduino IDE sketch for ESP32-S3 in C. Originally, I wasn't getting good performance in the microcontroller Pythons. I overcame that hurdle, so I focused more on the Python implementation. I need to clean up the code and will also release it.

**Comment by shymega** [comment]
There is a clear 'Discussions' tab, where this should go. It's very clear both in the Issue flow and in the tabs, that issues are _only for bugs_.

I will convert it now. But please pay attention to the flow when you create an issue, it saves the maintainers our time.

**Reply by jzhvymetal** [reply]
Thanks and Sorry I missed the Discussions tab.  Modified the post to remove the bug info.
